docs/alert-routing.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: Alert Routing
+permalink: /alert-routing
+---
+
+
```

Comment:
@nadir-akhtar-coinbase If you have bandwidth it'd be amazing if we could get your eyes on this file!